packages/metals-vscode/src/testExplorer/testRunHandler.ts

```diff
@@ -110,9 +110,11 @@ export async function runHandler(
         tests: [],
       };
     } else {
+      const { id, parent, _metalsParent } = test;
+      const idStartsWithParent = parent && id.startsWith(`${parent.id}.`);
       return {
-        className: test._metalsParent.id,
-        tests: [test.label],
+        className: _metalsParent.id,
+        tests: [idStartsWithParent ? id.slice(parent.id.length + 1) : id],
```

Comment:
When do we encode the test with name of the parent? Could we fix that on the server side?

Reply:
Do we still need the change here?

Reply:
Yeah, I asked you again for review kind of by accident (I revoked  the request but I guess it already alarmed you). I forgot for a sec I was supposed to fixed that. :D